adm_program/languages/en.xml

```diff
@@ -1396,19 +1396,19 @@
   <string name="SYS_SSO_SAML_ENABLED_DESC">This setting turns Admidio into a SAML 2.0 identity provider (idP) for single-sign-on, which allows other web applications to authenticate users agains the admidio user list.</string>
   <string name="SYS_SSO_SAML_ENTITY_ID">SAML Entity Id</string>
   <string name="SYS_SSO_SAML_ENTITY_ID_DESC">The unique identifier for SAML of this Admidio instance. Typically this is the URL of your installation. The client needs to be set up with the exact same entity ID, otherwise login will not work.</string>
-  <string name="SYS_SSO_SAML_STATIC_SETTINGS">IdP metadata for client setup</string>
+  <string name="SYS_SSO_STATIC_SETTINGS">IdP metadata for client setup</string>
   <string name="SYS_SSO_SAML_SSO_ENDPOINT">SSO Endpoint</string>
   <string name="SYS_SSO_SAML_SLO_ENDPOINT">SLO Endpoint</string>
   <string name="SYS_SSO_SAML_METADATA_URL">Metadata URL</string>
   <string name="SYS_SSO_SAML_SUPPORTED_PROTOCOLS">Supported Protocols</string>
-  <string name="SYS_SSO_SAML_SIGNING_KEY">Key for signatures</string>
-  <string name="SYS_SSO_SAML_ENCRYPTION_KEY">Key for encryption</string>
+  <string name="SYS_SSO_SIGNING_KEY">Key for signatures</string>
+  <string name="SYS_SSO_ENCRYPTION_KEY">Key for encryption</string>
   <string name="SYS_SSO_SAML_ROLES">Included groups/roles</string>
   <string name="SYS_SSO_SAML_ROLES_DESC">Select all roles that should be transmitted upon successful login to the Service Provider. As many clients use the transmitted role verbatim, one can map Adimidio's role names to role names desired on the client / Service Provider's side. If the SAML role is left empty, Admidio's role name is used instead.</string>
-  <string name="SYS_SSO_SAML_ATTRIBUTES">Included user attributes</string>
-  <string name="SYS_SSO_SAML_ATTRIBUTES_DESC">Select all user fields that should be transmitted upon successful login to the Service Provider. For each field, the mapping to the SAML attribute name can be given. If the SAML column is left empty, Admidio's field name will be used instead.</string>
-  <string name="SYS_SSO_SAML_ATTRIBUTE">SAML attribute name</string>
-  <string name="SYS_SSO_SAML_ATTRIBUTES_ALLOTHER">Include all remaining fields with Admidio's internal field name</string>
+  <string name="SYS_SSO_ATTRIBUTES">Included user attributes</string>
```

Comment:
@kainhofer In general it would be nice if not all translation strings have the prefix SSO or SAML because the strings should also be used in other scenarios. e.g. SYS_INCLUDED_USER_ATTRIBUTES ..

Reply:
In this particular case, the string came from a SAML-specific text, which I tried to generalize also for the OIDC SSO. On the other hand, the _DESC of the string is SSO-specific, so I didn't want to tear the string and the corresponding description apart name-wise. In other cases I added generic string IDs *SYS_ADD_ALL_FIELDS", "SYS_ADD_ROLE", "SYS_ADD_ALL_ROLES", "SYS_LOGIN_TO", "SYS_ERROR_UUID_MISSING", etc